benchmarks/mv_prod.cpp

```diff
@@ -0,0 +1,54 @@
+#include <boost/numeric/ublas/vector.hpp>
+#include <boost/numeric/ublas/matrix.hpp>
+#include <boost/program_options.hpp>
+#include "prod.hpp"
+#include <complex>
+#include <string>
+
+namespace po = boost::program_options;
+
+template <typename T>
+void benchmark(std::string const &type)
+{
+  using matrix = ublas::matrix<T, ublas::basic_row_major<>>;
+  using vector = ublas::vector<T>;
+  prod<vector(matrix, vector)> p("prod(matrix<" + type + ">, vector<" + type + ">)");
+  p.run(std::vector<long>({1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096}));
+}
+
+int main(int argc, char **argv)
```

Comment:
I think there are too many of the same code lines are in [mm_prod](https://github.com/boostorg/ublas/pull/57/files#diff-25c518be5ef2d6f19157cebf394e276f). Do we then have to repeat all the lines for all new benchmarks? Lets try to stick to DRY.

Reply:
Please elaborate.

Reply:
The main of mm_prod and mv_prod are almost the same. I thought that we can do better if we don't repeat ourselves. See my gist [example](https://gist.github.com/bassoy/96b03bd9204758a3f4763ee8a212b3c1).